gui/slick/interfaces/default/home.mako

```diff
@@ -222,7 +224,7 @@ $(document).ready(function(){
             4: { sorter: 'quality' },
             5: { sorter: 'eps' },
             % if sickbeard.FILTER_ROW:
-            6: { filter : 'parsed' }
+                7: { filter : 'parsed' }
             % endif
         },
```

Comment:
No, because column 6 is the show size, and I'm just using an integer ([see here](https://github.com/SiCKRAGETV/SickRage/pull/2356/files#diff-e00affd852c5d93c6d5e2d8e92320048R216)). So I don't need to provide a sorting method.